spec/features/goal_tool_spec.rb

```diff
@@ -15,7 +15,7 @@
   end
 
   scenario "Displays due date" do
-    expect(page).to have_content "Due: #{Date.today.to_s(:date)}"
+    expect(page).to have_content "Due: #{Date.today.strftime("%b %d %Y")}"
```

Comment:
The only reason that I recommended this is to alleviate the problem of having to update the spec if the date configuration changes. But if you feel that the requirement won't change, I'm fine with the way it is written.